documentation/airport-guides/lszh.md

```diff
@@ -0,0 +1,51 @@
+### About this facility
+Skyguide Air Navigation Center, Dübendorf, Switzerland
+269,160 annual operations in 2016
+
+Located at Dübendorf Air Base (close to Zurich), besides the LSAZ FIR, the Skyguide Air Navigation Center is also responsible for the in- and outbound traffic of Switzerlands major airport Zurich (LSZH).
+
+### Airspace
+The airspace area extends up to 55nm of distance to the airport and is also covering parts of German national territory. Due to the fact that approaches to runway 14 and 16 require the use of German airspace, these approaches aren't available during the German ordinance period. (MON - FRI: 2100-0700LT; SAT, SUN, holidays;  2000-0700LT). Therefore runway 28 or 34 is usually used for landings then which is reducing the capacity of the airport. Vertically the airspace is covering aircraft up to FL190.
+
+### Instrument Procedures
+For outbound traffic LSZH offers a couple of SIDs eventhough usually only DEGES and VEBIT are used. Inbound traffic follows a STAR to either GIPOL or AMIKI. Inbound traffic must be vectored to their final approach course.
```

Comment:
Phrasing:
`LSZH offers several departure procedures, however the DEGES and VEBIT departures are the most commonly used. Arrivals follow their STAR's to either GIPOL or AMIKI, where aircraft must be vectored to their final approach course.`